### PR TITLE
feat(dashboards): Update Widget Builder doc to include Big Number Equations

### PR DESCRIPTION
**Pull request by harshithadurai**
## DESCRIBE YOUR PR
Update Widget Builder doc to include new Equations feature for Big Number Widgets.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Comment by harshithadurai** [comment]
#https://github.com/getsentry/sentry-docs/blob/42d449d0f6e5597993ccd5f5f00d4d5385c7d000/docs/product/dashboards/widget-builder/index.mdx#choose-what-to-plot-in-the-y-axis

Don't think I need to specifically mention equations for Big Number here since this section already talks about equations in a general sense that also applies well to Big Numbers

**Comment by narsaynorath** [comment]
QQ, I see this is set to Draft so I just wanted to check, are you planning to update the general guidelines of adding an equation in this PR as well?

**Review by narsaynorath — APPROVED** [review]
lgtm, let's wait until we start releasing to EA (tomorrow) to merge this

**Review by coolguyzone — APPROVED** [review]

**Pull request merged**
